src/prompt.rs

```diff
@@ -201,58 +201,41 @@ impl Prompt {
         // needed.
         let safe_prompt = || SAFE_PROMPT.to_string();
 
-        // In-memory cursor using a vector for data.
-        let cursor = Cursor::new(Vec::new());
-
-        let t = term::TerminfoTerminal::new(cursor);
-        if t.is_none() {
-            return safe_prompt();
-        }
-        let mut t = t.unwrap();
+        let bufwtr = BufferWriter::stderr(ColorChoice::Always);
```

Comment:
What exactly does the `ColorChoice::Always` mean here?

Reply:
Ah:
> Try very hard to emit colors. This includes emitting ANSI colors on Windows if the console API is unavailable.

https://docs.rs/termcolor/1.0.5/termcolor/enum.ColorChoice.html#variant.Always